nihms-integration/src/test/java/org/dataconservancy/pass/deposit/builder/fedora/FcrepoModelBuilderIT.java

```diff
@@ -79,19 +99,75 @@ public void testElementValues() {
 
         // Cannot compare ID strings, as they change when uploading to a Fedora server.
         Publication publication = (Publication)entities.get(submissionEntity.getPublication());
-        assertEquals(submission.getMetadata().getArticleMetadata().getDoi().toString(), publication.getDoi());
+        assertEquals(EXPECTED_DOI, submission.getMetadata().getArticleMetadata().getDoi().toString());
 
         assertNotNull(submission.getFiles());
         assertEquals(2, submission.getFiles().size());
 
         // Confirm that some values were set correctly from the Submission metadata
         DepositMetadata.Journal journalMetadata = submission.getMetadata().getJournalMetadata();
-        assertEquals("Food Funct.", journalMetadata.getJournalTitle());
-        assertEquals("TD452689", journalMetadata.getJournalId());
-        assertEquals("2042-6496,2042-650X", journalMetadata.getIssn());
+        assertEquals(EXPECTED_JOURNAL_TITLE, journalMetadata.getJournalTitle());
+        assertEquals(EXPECTED_ISSN, journalMetadata.getIssn());
 
         DepositMetadata.Manuscript manuscriptMetadata = submission.getMetadata().getManuscriptMetadata();
-        assertEquals("http://dx.doi.org/10.1039/c7fo01251a", manuscriptMetadata.getManuscriptUrl().toString());
+        assertNull(manuscriptMetadata.getManuscriptUrl());
+
+//        assertTrue(submission.getMetadata().getArticleMetadata().getUnderEmbargo());
+        assertEquals(EXPECTED_EMBARGO_END_DATE, submission.getMetadata().getArticleMetadata().getEmbargoLiftDate()
+                .format(DateTimeFormatter.ofPattern("uuuu-MM-dd")));
+
+        List<DepositMetadata.Person> persons = submission.getMetadata().getPersons();
+        int authors = 0;
+        int pis = 0;
+        int copis = 0;
+        int submitters = 0;
+        for (DepositMetadata.Person person : persons) {
+            switch (person.getType()) {
+                case author:
+                    authors++;
+                    break;
+                case pi:
+                    pis++;
+                    break;
+                case copi:
+                    copis++;
+                    break;
+                case submitter:
+                    submitters++;
+                    break;
+            }
+        }
+
+        assertEquals(EXPECTED_SUBMITER_COUNT, submitters);
+        assertEquals(EXPECTED_PI_COUNT, pis);
+        assertEquals(EXPECTED_CO_PI_COUNT, copis);
+        assertEquals(EXPECTED_AUTHOR_COUNT, authors);
+
+        assertTrue(persons.stream()
```

Comment:
These additional tests are nice.  Do you want to include them in FilesystemModelBuilderTest as well?  I don't know what the right thing to do is regarding having complete tests for two very similar functionalities versus avoiding redundant tests.

Reply:
@Trumbore I went ahead and added these to `FilesystemModelBuilderTest` and `FcrepoModelBuilderIT`.

I think that this functionality is critical enough that this duplication is "ok".  Maybe not "warranted", but definitely "ok".  I wanted my changes to the two tests to be minimal, so I just added these in to insure coverage.